share/misc/bsd-family-tree

```diff
@@ -454,32 +454,32 @@ FreeBSD 14 -current   |  NetBSD -current   OpenBSD -current    DragonFly -curren
 Time
 ----------------
 
-Time tolerance +/- 6 months, depending on which book/article you read; if it
-was the announcement in Usenet or if it was available as tape.
+Time tolerance +/- 6 months, depending on which book/article you read; if it was
```

Comment:
```
Time tolerance on 4.3BSD and earlier releases can be as large as 6 months due to the uncertainty of historical records; the early release to special groups; and the slow speed of tape duplication at the time.
Different articles or books the period will have different dates since they relied on different dates: newest file in the distribution vs public announcement vs first available tape.```
might be a little more accurate.

Reply:
> … articles or books the period …

_of_, _from_ or _for_ the period?

Assuming _from_ (because _relied_ is past tense): I rewrote the sentence a little, to not use _different_ in three places. Also a couple of tweaks to the first sentence. 

[`9779ac5` (#812)](https://github.com/freebsd/freebsd-src/pull/812/commits/9779ac5e4d3ef99c6ae487eecce0d4d8b302e2a9) – let me know your thoughts, thanks.

Reply:
Shall we go with 'articles or books _from_ the period'?